### PR TITLE
UI enhancements

### PR DESCRIPTION
**Pull request by Thejasri1**
Changes made.
1. accordion
2. spinners
3. cursor pointer 
4. userAgent changed

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -144,7 +154,7 @@ app.post("/", async (req, res) => {
       console.log(e);
     }
 
-    //after executing the all the process it will redirect it home page :
+    // //after executing the all the process it will redirect it home page :
```
Comment (priyankcommits) [inline index.js]: why 4 slashes here?

File: index.js [hunk 2]
```diff
@@ -76,6 +71,21 @@ app.post("/", async (req, res) => {
     }
   };
   await getAddressByReverseCoding();
+  //IP Address :
+  const clientIp = requestIp.getClientIp(req);
+  //userAgent :
+  const userAgentMobileOrDesktop =
+    req.useragent.isMobile === true ? "isMobile : true" : "isDesktop : true";
```
Comment (priyankcommits) [inline index.js]: req.useragent.isMobile === true ? "Mobile" : "Desktop";

Anything that user will read should be in presentable format

File: views/punchIn.js [hunk 3]
```diff
@@ -1,5 +1,29 @@
...
+    if (panel.style.display === "block") {
+      panel.style.display = "none";
+    } else {
+      panel.style.display = "block";
+    }
+  });
+}
+
+function spinner() {
+  console.log("in here");
```
Comment (priyankcommits) [inline views/punchIn.js]: dont push console log statements, they are only for debugging purposes and printing catch block errors

File: views/punchIn.pug [hunk 4]
```diff
@@ -11,6 +12,46 @@ html
 	body
 		div(class="logo-container")
 			img(src="https://www.thejavascriptshop.com/assets/banner1.png" class="logo")
+		button(class="accordion")
+			svg(xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-info-circle" viewBox="0 0 16 16")
+				path(d="M8 15A7 7 0 1 1 8 1a7 7 0 0 1 0 14zm0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16z")
+				path(d="m8.93 6.588-2.29.287-.082.38.45.083c.294.07.352.176.288.469l-.738 3.468c-.194.897.105 1.319.808 1.319.545 0 1.178-.252 1.465-.598l.088-.416c-.2.176-.492.246-.686.246-.275 0-.375-.193-.304-.533L8.93 6.588zM9 4.5a1 1 0 1 1-2 0 1 1 0 0 1 2 0z")
+			p(class="info-caption") click the icon for id's
```
Comment (priyankcommits) [inline views/punchIn.pug]: Presentable format: Click the Icon for ID's

File: package.json [hunk 5]
```diff
@@ -7,6 +7,7 @@
     "@slack/socket-mode": "^1.2.0",
     "@slack/web-api": "^6.7.1",
     "axios": "^0.27.2",
+    "bootstrap-icons": "^1.8.1",
```
Comment (priyankcommits) [inline package.json]: remove everything related to bootstrap icons, its a very big package and for a very small thing we should not install a big package.

**Review by priyankcommits — CHANGES_REQUESTED** [review]

File: package.json (hunk 5 above)
Comment (Thejasri1) [reply package.json]: for icon can i use font awesome instead of bootstrap.

File: views/punchIn.pug [hunk 6]
```diff
@@ -11,6 +12,44 @@ html
 	body
 		div(class="logo-container")
 			img(src="https://www.thejavascriptshop.com/assets/banner1.png" class="logo")
```
Comment (priyankcommits) [inline views/punchIn.pug:13]: Don't link images from external websites. Download the image and use it from local.

File: views/punchIn.pug [hunk 7]
```diff
@@ -11,6 +12,44 @@ html
...
+		button(class="accordion")
+			img(src="https://www.pngitem.com/pimgs/m/195-1951784_info-icon-svg-transparent-background-information-icon-hd.png" class="icon-for-info")
```
Comment (priyankcommits) [inline views/punchIn.pug]: Don't link images from external websites. Download the image and use it from local.

**Review by priyankcommits — CHANGES_REQUESTED** [review]

**Pull request merged**